src/Application/AnalyzeCommand.php

```diff
@@ -4,20 +4,33 @@
 
 namespace TwigStan\Application;
 
-use InvalidArgumentException;
+use Nette\Neon\Neon;
+use PHPStan\Analyser\Error;
+use PHPStan\Analyser\Ignore\IgnoredErrorHelper;
+use PHPStan\File\CouldNotWriteFileException;
+use PHPStan\File\FileReader;
+use PHPStan\File\FileWriter;
+use PHPStan\File\ParentDirectoryRelativePathHelper;
+use PHPStan\Internal\DirectoryCreator;
+use PHPStan\Internal\DirectoryCreatorException;
+use PHPStan\ShouldNotHappenException;
```

Comment:
We should not depend on PHPStan for this.

Reply:
Removed all phpstan dependencies